tools/decrypt-hybrid-assets/README.md

```diff
@@ -0,0 +1,83 @@
+# Decrypt Hybrid Assets
+
+A rescue utility to decrypt proxies and shared flows taken directly from
+an apigee-runtime pod in an encrypted form.
+
+## Description
+
+Image, your trial Apigee hybrid organisation was deleted. The only thing that's
+left is a running runtime.
+
+No need to panic.
+
+## To take a backup snapshot
+
+1. Log into any apigee-runtime pod
+
+1. Read the value of an $CONTRACT_ENCRYPT_KEY_PATH environment variable
+
+    ```sh
+    echo $CONTRACT_ENCRYPT_KEY_PATH
+    ```
+
+    It points to a file that contains base64-encoded artefact encryption key.
+
+    For 1.3.x and 1.4.x it will be `/etc/encryption/plainTextDEK`
+
+1. Read the encoded key value
+
+    ```sh
+    cat $CONTRACT_ENCRYPT_KEY_PATH
+    ```
+
+1. tar/gz contents of the `/opt/apigee/apigee-runtime/data/$XXX/config/` directory.
+
+## To recover assets
+
+1. Clone the utility and add its directory to a PATH
```

Comment:
leave this as it is for now, but in future we will store this utility in the "Releases" in github